tests/unit_tests/test_make_indices.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <common/make_indices.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace cpp_bindgen;
+
+static_assert(std::is_same<integer_sequence<int>::value_type, int>::value, "");
+static_assert(integer_sequence<int, 1, 2, 3>::size() == 3, "");
+
+static_assert(std::is_same<make_integer_sequence<int, 3>, integer_sequence<int, 0, 1, 2>>::value, "");
+static_assert(std::is_same<make_integer_sequence<bool, 1>, integer_sequence<bool, false>>::value, "");
```

Comment:
```suggestion
```
I would like to remove this... GCC doesn't like it (because `false + 1` is kind of stupid).